packages/create/test/unit/cli/add/change-data-source.test.ts

```diff
@@ -0,0 +1,240 @@
+import { Command } from 'commander';
+import type { Editor } from 'mem-fs-editor';
+import type { ToolsLogger } from '@sap-ux/logger';
+import { addChangeDataSourceCommand } from '../../../../src/cli/add/change-data-source';
+import * as tracer from '../../../../src/tracing/trace';
+import * as common from '../../../../src/common';
+import * as logger from '../../../../src/tracing/logger';
+import * as adp from '@sap-ux/adp-tooling';
+import type { CustomMiddleware } from '@sap-ux/ui5-config';
+import { UI5Config } from '@sap-ux/ui5-config';
+import * as mockFs from 'fs';
+import { join } from 'path';
+import * as prompts from 'prompts';
+
+const appManifest = jest
+    .requireActual('fs')
+    .readFileSync(join(__dirname, '../../../fixtures/adaptation-project', 'manifest.json'), 'utf-8');
+const descriptorVariant = JSON.parse(
+    jest
+        .requireActual('fs')
+        .readFileSync(join(__dirname, '../../../fixtures/adaptation-project', 'manifest.appdescr_variant'), 'utf-8')
+);
+
+jest.mock('fs');
+jest.mock('prompts');
+
+const abapServicesMock = {
+    getManifestUrl: jest.fn().mockResolvedValue('https://sap.example'),
+    getManifest: jest.fn().mockResolvedValue(JSON.parse(appManifest))
+};
+
+jest.mock('@sap-ux/system-access', () => {
+    return {
+        ...jest.requireActual('@sap-ux/system-access'),
+        createAbapServiceProvider: () => {
+            return {
+                getAppIndex: jest.fn().mockReturnValue({
+                    getManifestUrl: abapServicesMock.getManifestUrl
+                }),
+                getLayeredRepository: jest.fn().mockReturnValue({
+                    getManifest: abapServicesMock.getManifest
+                })
+            };
+        }
+    };
+});
+
+describe('adp-add/change-data-source', () => {
+    // Mocks
```

Comment:
All those comments are unneeded below, just leave empty space

Reply:
It seems that similar comments are used in some of the tests (e.g., https://github.com/SAP/open-ux-tools/blob/567c715072a4e0d81163bf8c7cd1fe000f222011/packages/create/test/unit/cli/generate/adaptation-project.test.ts#L32). Regardless, the comments are removed from our tests with: 01ea9ff8d5b8b50622d2cb9c299a9560cded6c15